pypesto/visualize/misc.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+import warnings
+from .clust_color import assign_colors
+from .clust_color import assign_colors_for_result_list
+
+
+def handle_result_list(results, colors=None, legends=None):
```

Comment:
might be worthwhile to rename these to process, as `handle` might also refer to a smart-pointer object:
https://wiki.python.org/moin/boost.python/handle (I know we should fix this in AMICI at some point aswell)

Reply:
Fine more me. For the other one I would have expected result_list_handle